src/resources/appointment.ts

```diff
@@ -367,6 +377,10 @@ export default class Appointment extends Conditional implements AppointmentResou
         params.data.attributes.staff_id = this.filters.user;
       }
 
+      if (this.filters.users) {
+        params.data.attributes.users = this.filters.users;
```

Comment:
This is the only thing I'm questioning at the moment. The request payload looks a little odd with this addition in my opinion.

```json
{
   "data":{
      "type":"appointments",
      "attributes":{
         "location_id":"1",
         "service_id":"1",
         "staff_id":"1",
         "users": [1, 2],
         "start":"2017-03-01 12:00:00"
      }
   }
}
```

Is there anything we can do to make it a little more consistent? Other than this, looks good to me

Reply:
Isn't it the same as with multiple services? We technically allow multiple service ids.

Reply:
Yes, which also looks odd. But perhaps that's what we can aim for here? What about "additional_staff_id" (for consistent naming) and it can accept an array of integers?

Reply:
Yes, that will be better. I'll change the name